app/controllers/api/jobs_controller.rb

```diff
@@ -4,7 +4,8 @@ class JobsController < Api::ApiController
     before_action :set_job, only: [:show, :retry, :update]
 
     def index
-      @jobs = Job.where(application_id: current_application.id)
+      @jobs = Job.order('created_at DESC').page(params[:page]).per(params[:per_page])
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.